fields/types/html/HtmlField.js

```diff
@@ -53,7 +53,7 @@ module.exports = Field.create({
 		};
 
 		this._currentValue = this.props.value;
-		tinymce.init(opts);
+		tinymce.init({ ...opts, branding: false });
```

Comment:
I hate to say this in 2019, but this will fail in our CI. Could you use `Object.assign` instead please?

Reply:
sure